ssl/record/ssl3_record_tls13.c

```diff
@@ -21,7 +21,7 @@
  *   -1: if the record's AEAD-authenticator is invalid or, if sending,
  *       an internal error occurred.
  */
-int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int send)
+int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int sending)
```

Comment:
I believe those may have been other functions that had shadow issues surrounding the use of the term |send|. That being said, in the end variables and/or functions are just addresses, and a dumb linker doesn't differentiate. Is it really a bad thing to avoid shadowing, regardless of the type?